landing-page.html

```diff
@@ -12,17 +12,17 @@
 
 {% comment %} Use the latest coding challenge if a stream hasn't been scheduled yet. {% endcomment %}
 {% if nextStream == null %}
-  {% assign offset = 1 %}
```

Comment:
I think your prettifier wasn't happy with my formatting... 😭 We should keep it indented for better readability!